### PR TITLE
Added a section to the CONTRIBUTING.md about opening the project

### PR DESCRIPTION
**Pull request by abbeycode**
This addresses a concern raised in Issue #564

### CODE REVIEW

**Comment by jspahrsummers** [comment]
Thanks! Can this suggest running [`script/bootstrap`](https://github.com/Carthage/Carthage/blob/dbf4fc70b4dc9312af92a6c523954d25b8c98d69/script/bootstrap) instead, though?

Carthage doesn't require Carthage to bootstrap its own repository, because that would make it really hard to get started with it in the first place. :wink:

**Comment by abbeycode** [comment]
@jspahrsummers Good call. I updated the pull request branch

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,6 +14,12 @@ If you can’t make the change yourself, please open an issue after making sure
 
 Unless you’re specifically improving something about the command-line experience of Carthage, please make code changes to [CarthageKit](README.md#carthagekit). This framework increases modularity, and allows other tools to integrate with Carthage more easily.
 
+## Get started
+
+After checkout, you can run the following command from the cloned directory, and then open the workspace in Xcode:
+
+    ./script/carthage bootstrap --platform mac --no-use-binaries
```
Comment (mdiep) [inline CONTRIBUTING.md]: As @jspahrsummers pointed out above, this can just be `./script/bootstrap`. :smile:

Comment (abbeycode) [inline CONTRIBUTING.md]: Whoops, I misunderstood. It's corrected for real now

**Comment by mdiep** [comment]
Thanks! :sparkles:

**Pull request merged**